2200094/10505블균형.py

```diff
@@ -0,0 +1,13 @@
+T = int(input())
+
+
+for t in range(1, T+1):
+    cnt = int(input())
+    money = list(map(int, input().split()))
+    avg = int(sum(money)/len(money))
```

Comment:
깔끔하네요! 👍 
```suggestion
    avg = sum(money)/len(money)
```
정확한 평균 값을 구하려면 `int` 연산을 사용하지 않는 게 맞습니다! 😄